docs/keto/index.md

```diff
@@ -1,45 +1,56 @@
 ---
 id: index
-title: Introduction
+title: Introduction to Ory Permissions
+sidebar_label: Introduction
 ---
 
-Ory Permissions (based on the open-source Ory Keto Permission Server) is the first open-source implementation of
-[Zanzibar: Google's Consistent, Global Authorization System](https://research.google/pubs/pub48190/).
+# Introduction
 
-With Ory Permissions, you can
+Ory Permissions is a modern permission system you can use to create a complete authorization system for your application or
+website, no matter the size or the complexity of the ACLs (access-control lists) your use case requires. Ory Permissions, based on
+the open-source Ory Keto Permission Server, is the first open-source implementation of the design principles and specifications
+described in [Zanzibar: Google's Consistent, Global Authorization System](https://research.google/pubs/pub48190/).
 
-- unify authorization logic in one service that is the authoritative source across all your applications.
-- define a permission model such as RBAC, ABAC, or a more flexible model that fits your use case.
-- work independent of any tech-stack, with [SDKs](./sdk/01_overview.md) available for all major programming languages.
-- query permissions across the world against the globally-distributed Ory Network.
-- issue fine-grained permissions, for example `user x can read document y`.
-- inherit permissions through groups, roles, hierarchies, and more.
+Implementing the design principles described in the Zanzibar paper allows Ory Permissions to offer the following benefits over
+older, widely adopted permission systems:
 
-## How Ory Permissions works
+- Flexibility: Older permission systems typically rely on a fixed set of permissions and access control rules that are difficult
+  to customize or modify. Ory Permissions provides a flexible data model you can customize to fit a wide range of use cases.
+  Creating custom permission models is aided by Ory Permission Language, a developer-friendly, TypeScript-based configuration
+  language.
 
-### Relationships
+- Scalability: While traditional permission systems often struggle to handle large-scale applications and services with high
+  request volume, Ory Permissions is designed to be highly scalable and handle a large number of concurrent requests without
+  sacrificing performance or reliability.
 
-Ory Permissions operates on and manages relationships. These relationships represent the source of truth from which all
-permissions are derived. Common examples include:
+- Consistency: In older permissions systems, the data is often eventually consistent, which leaves windows of opportunity for
+  potential unauthorized access to resources before the data is correctly propagated to all parts of the system. By implementing
+  Zanzibar, Ory Permissions puts consistency in the spotlight and ensures that permission checks that are at the critical path of
+  user experience, are always answered correctly.
```

Comment:
Without Zookies, we also only have eventual consistency. So we can't really claim that yet.